### ISSUE TITLE
[BUG] Zwift Play not working with Rouvy on Win11 and QZ on Android v2.18.8 

### ISSUE THREAD

**Issue by PandaboyBSRC** [cagnulein/qdomyos-zwift#2896]
**Describe the bug**
When QZ is running and has Zwift Ride play controllers and Wahoo Kickr connected, Zwift play buttons are not working in Rouvy. Also I tried to do a Royvy workout and it was not detecting when I was pedalling

**To Reproduce**
1. Turn on Wahoo Kickr and Zwift Play controllers
2. Start QZ app on Android phone
3. Start Rouvy
4. Ensure Rouvy is connected to phone for power, cadence and controllable
5. Start workout
6. Nothing happens when pressing any of the Zwift play buttons - gears increase on QZ app but make no difference to Royvy

**Expected behavior**
Buttons should do something in Rouvy

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 11
 

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S24 Ultra
 - OS: Android
 - Version: 14

**Append a debug log**
[debug-Tue_Dec_10_11_04_17_2024.log](https://github.com/user-attachments/files/18078405/debug-Tue_Dec_10_11_04_17_2024.log)

 Follow this guide https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
  
  
**Additional context**

**Comment by cagnulein** [comment]
hi remove the trainer from the ftms bike settings and press ok and restart
qz
let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 10 dic 2024 alle 12:18 PandaboyBSRC ***@***.***>
ha scritto:

**Comment by PandaboyBSRC** [comment]
I don't know where "ftms bike settings" are? I removed the trainer from "manual device" under "advanced settings". Gears and other Zwift play buttons do not seem to do anything. When I press the gear buttons the gear goes up in the QZ app (all the way to 100 or more!)  but it makes no difference to how hard it is to pedal

I have now added an Ant+ dongle to my machine but that does not seem to have made any difference either.

New debug log here:
[debug-Tue_Dec_10_15_22_26_2024.log](https://github.com/user-attachments/files/18082022/debug-Tue_Dec_10_15_22_26_2024.log)

**Comment by cagnulein** [comment]
it's under the bike options in QZ.

**Comment by PandaboyBSRC** [comment]
Ok, have now found FTMS Bike and disabled it. Will try again...

**Comment by cagnulein** [comment]
ok let me know!

**Comment by PandaboyBSRC** [comment]
Nope, Zwift Play buttons still have no effect in Rouvy. Changing gear does nothing and none of the other buttons do anything. 

New log attached
[debug-Tue_Dec_10_16_03_05_2024.log](https://github.com/user-attachments/files/18082569/debug-Tue_Dec_10_16_03_05_2024.log)

**Comment by cagnulein** [comment]
that's strange, you can try with ftms bike enabled back to the trainer and enable the gears zwift experimental setting and restart qz and the trainer.

but for sure without the gears zwift experimental you have to feel the difference on the trainer in terms of resistance. let me know

**Comment by PandaboyBSRC** [comment]
Hi. i've been trying to troubleshoot the problem and have just tried using a Samsung Tab to run Rouvy instead of my laptop and it appeared to work perferctly, so the problem seems to be with my Windows laptop rather than your app.
